recipes/json-c/all/conanfile.py

```diff
@@ -70,3 +68,5 @@ def package_info(self):
         self.cpp_info.set_property("cmake_target_name", "json-c::json-c")
         self.cpp_info.set_property("pkg_config_name", "json-c")
         self.cpp_info.libs = collect_libs(self)
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.system_libs.append("m")
```

Comment:
It doesn't depend on libm AFAIK

Reply:
I found it here:
https://github.com/json-c/json-c/blob/d1716fe431694bfc08ad94d2e8d2a32d79b47fa3/CMakeLists.txt#LL98-L100C8

Reply:
https://github.com/json-c/json-c/pull/182